### PR TITLE
zh-CN locale files

### PR DESCRIPTION
**Pull request by yuxisteem**
Hi I just added zh-CN locale file(s) with Simplified Chinese translations. Let me know if there are any changes needed. Thanks.

### CODE REVIEW

**Comment by marcometz** [comment]
Thanks a lot for your work!

File: config/locales/active_admin.zh-CN.yml [hunk 1]
```diff
@@ -0,0 +1,645 @@
...
+      labels:
+        set_article_online: "设置文章为在线"
+        set_article_offline: "设置文章为离线"
+        set_menue_offline: "设置菜单为在线"
+        set_menue_online: "设置菜单为离线"
+        reset_cache: "清空缓存"
+        send_default_conf_mails: "发送确认电子邮件"
+        destroy: "删除"
+      button_label: "批量编辑"
+      delete_confirmation: "你确定要删除这些%{plural_model}吗？撤消选项不可用。" available."
```
Comment (donni106) [inline config/locales/active_admin.zh-CN.yml]: Something is wrong here!
`available."` should have been removed!
I will fix it on master.

File: config/locales/zh-CN.yml [hunk 2]
```diff
@@ -0,0 +1,632 @@
...
+      failure: "由于%{reason}，无法从%{kind} 认证。"
+      success: "成功从 %{kind} 认证帐户。"      
+    passwords:
+      no_token: "如果没有密码重置邮件无法访问这个页面。如果你是从密码重置邮件访问过来的，请确认输入的网址完整。"
+      send_instructions: "在几分钟内你会收到一封密码重置邮件。"
+      send_paranoid_instructions: "如果我们的数据库中保存有你的电子邮件地址，你会在几分钟内收到一封如何恢复密码的邮件。"
+      updated: "你的密码已被成功更改，你现在已经登录。"
+      updated_not_active: "你的密码已被成功更改。"      
+    registrations:
+      updated: "Your account has been updated successfully."
```
Comment (donni106) [inline config/locales/zh-CN.yml]: Here is a translation missing.

File: config/locales/active_admin.zh-CN.yml (hunk 1 above)
Comment (donni106) [reply config/locales/active_admin.zh-CN.yml]: --> https://github.com/ikuseiGmbH/Goldencobra/commit/5bc3536a0f8dda8e21e28d311ec9ba7084ae5e8e

File: config/locales/zh-CN.yml (hunk 2 above)
Comment (donni106) [reply config/locales/zh-CN.yml]: --> https://github.com/ikuseiGmbH/Goldencobra/commit/632e10be7448b238e28989423922d8b3467376aa

**Pull request merged**